### PR TITLE
zpaq: add page

### PR DESCRIPTION
**Pull request by dy-tea**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**
zpaq v7.15

### CODE REVIEW

File: pages/common/zpaq.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+> Incremental Journaling Backup Utility and Archiver.
+> More information: <https://mattmahoney.net/dc/zpaq.html>.
+
+- [a]dd a file or directory to a new or existing archive:
+
+`zpaq a {{path/to/archive.zpaq}} {{path/to/file_or_directory}}`
+
+- Create or add to encrypted archive with password X:
+
+`zpaq a -k{{X}} {{path/to/archive.zpaq}} {{path/to/file_or_directory}}`
```
Comment (spageektti) [inline pages/common/zpaq.md:12]:
```suggestion
- Create or add to encrypted archive:

`zpaq a -k{{password}} {{path/to/archive.zpaq}} {{path/to/file_or_directory}}`
```

File: pages/common/zpaq.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# zpaq
+
+> Incremental Journaling Backup Utility and Archiver.
```
Comment (gutjuri) [inline pages/common/zpaq.md:3]:
```suggestion
> Incremental journaling backup utility and archiver.
```

File: pages/common/zpaq.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://mattmahoney.net/dc/zpaq.html>.
+
+- [a]dd a file or directory to a new or existing archive:
+
+`zpaq a {{path/to/archive.zpaq}} {{path/to/file_or_directory}}`
+
+- Create or add to encrypted archive:
```
Comment (gutjuri) [inline pages/common/zpaq.md:10]:
```suggestion
- Create or add to an encrypted archive:
```

File: pages/common/zpaq.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`zpaq a -k{{password}} {{path/to/archive.zpaq}} {{path/to/file_or_directory}}`
+
+- E[x]tract the most recent recent versions of files:
```
Comment (gutjuri) [inline pages/common/zpaq.md:14]:
```suggestion
- E[x]tract the most recent versions of files:
```

File: pages/common/zpaq.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`zpaq x {{path/to/archive.zpaq}}`
+
+- [l]ist or compare external files to archive by dates:
```
Comment (gutjuri) [inline pages/common/zpaq.md:18]:
```suggestion
- [l]ist the archive contents:
```

I think this makes it clearer for non-advanced users.

File: pages/common/zpaq.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`zpaq l {{path/to/archive.zpaq}}`
+
+- Set the level of compression (higher means more compression but slower):
+
+`zpaq a {{path/to/archive.zpaq}} -m{{1|2|3|4|5}} {{path/to/file_or_directory}}`
+
+- E[x]tract to directory as of the date or version number:
+
+`zpaq x {{path/to/archive.zpaq}} {{path/to/extract}} -to {{path/to/output}} -until {{YYYY-MM-DD}}`
```
Comment (gutjuri) [inline pages/common/zpaq.md:28]:
```suggestion
- E[x]tract the specified files from the archive that are not newer than the specified date:

`zpaq x {{path/to/archive.zpaq}} {{path/in/archive/to/extract}} -to {{path/to/output}} -until {{YYYY-MM-DD}}`
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thanks, I have some suggestions!

File: pages/common/zpaq.md (hunk 4 above)
Comment (dy-tea) [reply pages/common/zpaq.md]: Wow I'm amazed I missed that

**Review by gutjuri — APPROVED** [review]
Thanks!

**Review by spageektti — APPROVED** [review]
Thanks for your contribution

**Pull request merged**